nixos/tests/nixos-test-driver/default.nix

```diff
@@ -0,0 +1,75 @@
+# All tests that don't need a specific config go here.
+
+{ pkgs, ... }:
+
+let
+  lib = pkgs.lib;
+  machines = import ./machines.nix;
+  mkTest =
+    name:
+    { script
+    , nodes ? { inherit (machines) basic; }
+    , enableOCR ? false
+    , skipTypeCheck ? false
+    }:
+    import ../make-test-python.nix
```

Comment:
This could be simplified with [`pkgs.testers.runNixOSTest`](https://nixos.org/manual/nixpkgs/unstable/#tester-runNixOSTest). Also evals faster because it reuses `pkgs`.